src/style.css

```diff
@@ -1 +1,153 @@
-/* add styles here */
+ @font-face {
+  font-family: Avenir-Book;
+  src: url('./fonts/Avenir-Book.ttf') format('truetype'),
+       url('./fonts/Avenir-Heavy.ttf') format('truetype');
```

Comment:
The `@font-face` rule specifies two different font files for the same `font-family` without differentiating them by `font-weight` or `font-style`. If these fonts are meant to represent different styles or weights, you should specify them accordingly to ensure the correct font is used.